make.cmd

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 rgbasm -omain.o main.s
-rgblink -p00 -omain.gb main.o
+rgbasm -owram.o wram.s
```

Comment:
Is this an alternative to something like `INCLUDE "hardware.inc"`?

Reply:
Including other ASM files into a single "master" one causes problems, and is basically bad practice.